src/watch/fsevents-importer.ts

```diff
@@ -1,8 +1,10 @@
-let fsEvents: typeof import('fsevents');
+let fsEvents: any; // TODO find better type solution, fsevents is only being installed on macOS
 let fsEventsImportError: Error | undefined;
 
+const mod = 'fsevents';
```

Comment:
If you need to extract it, why not use a helpful variable name?

Reply:
also pulled the variable into the function scope. probably doesn't matter much either way.